GTG/backends/__init__.py

```diff
@@ -67,10 +67,12 @@ def __init__(self):
                 # Something is wrong with this backend, skipping
                 log.warning("Backend %s could not be loaded: %r",
                             module_name, exception)
+                inactive_modules.append(module_name)
                 continue
             except Exception:
                 # Other exception log as errors
                 log.exception("Malformated backend %s:", module_name)
+                inactive_modules.append(module_name)
```

Comment:
Ditto. You need to use [`.add()`](https://docs.python.org/3.8/library/stdtypes.html#frozenset.add) since it is a set, not an list.